daemon/container_operations_unix.go

```diff
@@ -354,13 +354,18 @@ func killProcessDirectly(container *container.Container) error {
 	// In case there were some exceptions(e.g., state of zombie and D)
 	if process.Alive(pid) {
 		// Since we can not kill a zombie pid, add zombie check here
-		isZombie, err := process.Zombie(pid)
+		status, err := process.GetProcStatus(pid)
 		if err != nil {
 			logrus.WithError(err).WithField("container", container.ID).Warn("Container state is invalid")
 			return err
 		}
-		if isZombie {
+		if status != "" {
+			logrus.Warnf("container %s PID %d status is \"%s\"", container.ID, pid, status)
+		}
```

Comment:
Looks like this would print warnings in many cases (including "expected" ones);


From [PROC(5)](https://man7.org/linux/man-pages/man5/proc.5.html):

```markdown
(3) state  %c
     One of the following characters, indicating process
     state:

     R  Running

     S  Sleeping in an interruptible wait

     D  Waiting in uninterruptible disk sleep

     Z  Zombie

     T  Stopped (on a signal) or (before Linux 2.6.33)
        trace stopped

     t  Tracing stop (Linux 2.6.33 onward)

     W  Paging (only before Linux 2.6.0)

     X  Dead (from Linux 2.6.0 onward)

     x  Dead (Linux 2.6.33 to 3.13 only)

     K  Wakekill (Linux 2.6.33 to 3.13 only)

     W  Waking (Linux 2.6.33 to 3.13 only)

     P  Parked (Linux 3.9 to 3.13 only)
```

Reply:
how about 
```
		if status != "" && status != "R" && status != "S" {
			logrus.Warnf("container %s PID %d status is \"%s\"", container.ID, pid, status)
		}
```
or delete  logrus.Warnf